roles/pcc-api/README.md

```diff
@@ -0,0 +1,46 @@
+Role Name
+=========
+
+A brief description of the role goes here.
+
+Requirements
+------------
+
+Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+
+Role Variables
```

Comment:
update Role variables . put the required variables like the 
-> repo_mariadb should be true for the repo to be included